cmd/clusterctl/cmd/root.go

```diff
@@ -163,7 +164,9 @@ func initConfig() {
 		}
 	}
 
-	logf.SetLogger(logf.NewLogger(logf.WithThreshold(verbosity)))
+	log := logf.NewLogger(logf.WithThreshold(verbosity))
+	logf.SetLogger(log)
+	ctrl.SetLogger(log)
```

Comment:
Yes! the checks are working 😄

Reply:
Unfortunately missed it until after the release :) But yup worked.

I wonder in how many binaries / controllers this "panic" will pop up :D

Reply:
True, but it's doing what it's supposed to do 😄